packages/directlinespeech/src/createAdapters.js

```diff
@@ -92,20 +92,17 @@ export default async function create({
 
   // Supported options can be found in DialogConnectorFactory.js.
 
+  // Setting the language use for recognition.
   config.setProperty(PropertyId.SpeechServiceConnection_RecoLanguage, speechRecognitionLanguage);
 
-  // The following code set the output format. But currently, none of the following works for setting detailed output format.
-  // We will leave these code commented until the Speech SDK support, possibly it in one of the way mentioned below.
+  // The following code set the output format.
+  // As advised by Speech team, this API may change in the future.
+  config.setProperty(PropertyId.SpeechServiceResponse_OutputFormatOption, 'detailed');
 
-  // config.setProperty(PropertyId.SpeechServiceResponse_OutputFormatOption, OutputFormat[OutputFormat.Detailed]);
-  // config.setProperty(PropertyId.SpeechServiceResponse_RequestDetailedResultTrueFalse, true);
-  // config.setProperty(OutputFormatPropertyName, OutputFormat[OutputFormat.Detailed]);
-  // config.setServiceProperty(PropertyId.SpeechServiceResponse_RequestDetailedResultTrueFalse, "true", ServicePropertyChannel.UriQueryParameter);
-
-  // The following code is copied from C#, it should set from.id, but it did not.
-  // https://github.com/Azure-Samples/Cognitive-Services-Direct-Line-Speech-Client/blob/master/DLSpeechClient/MainWindow.xaml.cs#L236
+  // Setting the user ID for starting the conversation.
   userID && config.setProperty(PropertyId.Conversation_From_Id, userID);
 
+  // Setting Custom Speech and Custom Voice.
   // The following code is copied from C#, and it is not working yet.
```

Comment:
This is because the Speech SDK hasn't implemented the JavaScript yet, correct? I would clarify that here.

"Although this code is emulated from the C# Speech SDK, it is not working yet. The Speech SDK JavaScript has yet to be implemented."

Also, tracking? do we want to add the links into the code or no?

Reply:
Yes, because these functions are on C# and documented. But not on JavaScript.